controls/container_images.rb

```diff
@@ -55,71 +58,103 @@
   end
 end
 
-control 'cis-docker-benchmark-4.2' do
+control 'docker-4.2' do
   impact 1.0
   title 'Use trusted base images for containers'
-  desc 'Ensure that the container image is written either from scratch or is based on another established and trusted base image downloaded over a secure channel.'
-
-  tag 'daemon'
-  tag cis: 'docker:4.2'
-  tag level: 1
-  ref url: 'https://titanous.com/posts/docker-insecurity'
-  ref url: 'https://hub.docker.com/'
-  ref url: 'https://blog.docker.com/2014/10/docker-1-3-signed-images-process-injection-security-options-mac-shared-directories/'
-  ref url: 'https://github.com/docker/docker/issues/8093'
-  ref url: 'https://docs.docker.com/engine/reference/commandline/cli/#pull'
-  ref url: 'https://github.com/docker/docker/pull/11109'
-  ref url: 'https://blog.docker.com/2015/11/docker-trusted-registry-1-4/'
+  desc 'Ensure that the container image is written either from scratch or is based on another established and trusted base image downloaded over a secure channel.
+
+  Rationale: Official repositories are Docker images curated and optimized by the Docker community or the vendor. There could be other potentially unsafe public repositories. You should thus exercise a lot of caution when obtaining container images.'
+
+  tag 'docker'
+  tag 'cis-docker-benchmark-1.12.0:4.2'
+  tag 'cis-docker-benchmark-1.13.0:4.2'
+  tag 'level:1'
+  ref 'Docker Image Insecurity', url: 'https://titanous.com/posts/docker-insecurity'
+  ref 'Docker Hub', url: 'https://hub.docker.com/'
+  ref 'Docker 1.3: signed images, process injection, security options, Mac shared directories', url: 'https://blog.docker.com/2014/10/docker-1-3-signed-images-process-injection-security-options-mac-shared-directories/'
+  ref 'Proposal: Provenance step 1 - Transform images for validation and verification', url: 'https://github.com/docker/docker/issues/8093'
+  ref 'Use the Docker command line', url: 'https://docs.docker.com/engine/reference/commandline/cli/'
+  ref 'Add support for referring to images by digest', url: 'https://github.com/docker/docker/pull/11109'
+  ref 'Announcing Docker Trusted Registry 1.4 – New User Interface, Integrated Content Trust and Support for Docker Engine 1.9', url: 'https://blog.docker.com/2015/11/docker-trusted-registry-1-4/'
+
+  describe 'docker-test' do
+    skip 'Use trusted base images for containers'
```

Comment:
Is there any way that we can test that?

Reply:
Perhaps some spin on this? https://docs.docker.com/engine/security/trust/content_trust/#enable-and-disable-content-trust-per-shell-or-per-invocation

Could potentially check the `DOCKER_CONTENT_TRUST` env var value

Reply:
This could work

```
## Enable and disable content trust per-shell or per-invocation
In a shell, you can enable content trust by setting the DOCKER_CONTENT_TRUST environment variable. Enabling per-shell is useful because you can have one shell configured for trusted operations and another terminal shell for untrusted operations. You can also add this declaration to your shell profile to have it turned on always by default.

To enable content trust in a bash shell enter the following command:

export DOCKER_CONTENT_TRUST=1
```

Reply:
Great suggestion @grdnrio, I am opening a new ticket, so we can cover this in a separate PR

Reply:
thanks for the great feedback! @grdnrio can you have a look again? please

Reply:
Looks good 👍